### ISSUE TITLE
Allow different trading pair direction on external layer pointing to the same internal pair

### ISSUE THREAD

**Issue by kilrau** [ExchangeUnion/xud#893]
EDITED Proposal: Fixed base/quote currency direction xud internal, external can be any direction -> still maps to the same pair. This lets parties trade with each other no matter in which direction they added the trading pair. Improves UX and avoids splitting liquidity into two.

Needs to
* separate internal pair representation from external
* hardcoded internal representation for popular pairs
* map external representation to these
* separate pairs for non-hardcoded pairs

**Comment by sangaman** [comment]
This seems backwards to me. Virtually all exchanges and trading pairs I'm aware of use BTC as the quote currency. I don't think I've ever seen the price of BTC quoted in terms of how many LTC it can buy, for example.

Ultimately though I don't think we need to restrict the available trading pairs within xud, so users can have whatever they want for quote or base currencies. But for the defaults, keeping LTC/BTC makes sense to me and for any other pairs I think it makes more sense (and is more consistent with how things are typically quoted in the wild) to have the more valuable currency be the quote currency.

**Comment by offerm** [comment]
Agree.

I think we should restrict (at least for now)  BTC and WETH to be quote currencies and  allow everything else to be base currencies.

the pair WETH and BTC should only be WETH/BTC

Alternativly we can allow all currencies to be bases and quote but when we present data we do based on the the above logic. 

Or we allow the exchange to define it's pairs and present the book using whatever he defined.

**Comment by kilrau** [comment]
> Virtually all exchanges and trading pairs I'm aware of use BTC as the quote currency.

You are right of course, I got tricked by FIAT, including stable coins being quote currency on almost all exchanges and BTC/ETH the base. Crypto to Crypto the quote currency it's always the more valuable one - BTC or ETH. So agree on leaving the current pair as is.

>  I don't think we need to restrict the available trading pairs within xud, so users can have whatever they want for quote or base currencies.
> Alternativly we can allow all currencies to be bases and quote but when we present data we do based on the the above logic.

After the call I agree we want a "internal representation" of one pair in one particular direction but let the user/platform add pairs in any direction on the external layer. So indeed buy orders for one would become sell orders for another user @michael1011 . That's good UX. But I also agree it's not prio. Moved to post-1.0.0
